sources/packages/web/src/views/student/StudentAccountApplicationInProgress.vue

```diff
@@ -22,7 +22,7 @@
         ><v-img
           height="240"
           class="ml-2"
-          alt="An illustration of a women checking her phone. Illustration by Storyset."
+          alt="An illustration of a women checking her phone."
```

Comment:
We have 2 more vue files and few forms(form.io) having this  Illustration by Storyset in alt-tag.

![image](https://user-images.githubusercontent.com/54600590/228381911-1d9d003c-3f18-4abb-a029-e50569ffb545.png)

@hellolynn-tbtb to confirm is " Illustration by Storyset." needs to be part of alt-tag ?